SpatialGDK/Source/SpatialGDK/Public/SpatialGDKSettings.h

```diff
@@ -92,7 +112,11 @@ class SPATIALGDK_API USpatialGDKSettings : public UObject
 	UPROPERTY(EditAnywhere, config, Category = "Metrics", meta = (ConfigRestartRequired = false), DisplayName = "Metrics Report Rate (seconds)")
 	float MetricsReportRate;
 
-	/** Change 'Load' value in inspector to represent worker Frame Time instead of a fraction of target FPS.*/
+	/** 
+	* By default the SpatialOS Runtime reports server-worker instance’s load in frames per second (FPS). 
+	* Select this to switch so it reports as seconds per frame. 
```

Comment:
Is this not milliseconds per frame?

Reply:
according to this doc's description it's in seconds per frame

https://docs.google.com/document/d/1W2qxeW-mUxRXlHT5Xpy5TVYX2C4C_o-SP0vGuHtaJrE/edit

Reply:
Im going to confirm

Reply:
Confirmed in inspector